Framework/DataObjects/inc/MantidDataObjects/EventList.h

```diff
@@ -267,9 +267,9 @@ class MANTID_DATAOBJECTS_DLL EventList : public Mantid::API::IEventList {
 
   void filterByPulseTime(Types::Core::DateAndTime start, Types::Core::DateAndTime stop, EventList &output) const;
 
-  void filterByPulseTime(Kernel::TimeROI *timeRoi, EventList *output) const;
+  void filterByPulseTime(Kernel::TimeROI const *timeRoi, EventList *output) const;
```

Comment:
Just a question, why add const in front of *timeRoi? In the future, is it better to define all *timeRoi as a const?

Reply:
It was a suggestion made by cppcheck. `const Kernel::TimeROI *` and `Kernel::TimeROI const *` actually mean different things. [An article on the topic](https://www.geeksforgeeks.org/difference-between-constant-pointer-pointers-to-constant-and-constant-pointers-to-constants/).